jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -138,6 +138,7 @@ public HttpChannelState(ConnectionMetaData connectionMetaData)
     @Override
     public void initialize()
     {
+        // TODO this should be moved somewhere common?
```

Comment:
Leftover?

Reply:
No, it is a real new TODO.   This conversion of a collection of listeners into a single listener is not really uniq to HCS.   We need to review all Violation usage to make sure it is correct with regards to collections of listeners.